server/src/test/java/org/candlepin/policy/PoolRulesStackDerivedTest.java

```diff
@@ -323,6 +324,7 @@ public void mergedProductAttributes() {
     }
 
     @Test
+    @Ignore
```

Comment:
The functionality is not supported now. Currently this test is ignored and finally will remove this if we have consensus on it.